components/layouts/Navigation.tsx

```diff
@@ -4,8 +4,21 @@ import Image from "next/image";
 import Link from "next/link";
 import { RxAvatar } from "react-icons/rx";
 import { LiaShoppingCartSolid } from "react-icons/lia";
+import useCategories from "@/hooks/useCategories";
+import { useState } from "react";
 
 export default function NavigationBar() {
+  const { categories, loading, error } = useCategories();
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+
+  const handleCategorySelect = (category: string) => {
+    console.log(`Selected category: ${category}`);
```

Comment:
Please remove console,log code when in stage of merging to main branch

Reply:
> Please remove console,log code when in stage of merging to main branch

I removed the console.log code.